drivers/watchdog/wdt_mcux_wdog.c

```diff
@@ -0,0 +1,186 @@
+/*
+ *
+ * Copyright (c) 2018, NXP
+
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <watchdog.h>
+#include <clock_control.h>
+#include <fsl_wdog.h>
+
+#define SYS_LOG_LEVEL SYS_LOG_WDT_LEVEL
```

Comment:
Shouldn't it be `CONFIG_SYS_LOG_WDT_LEVEL`?

Reply:
Good catch, thanks. Fixed.